src/components/ReportActionItem/ReportActionItemImages.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import {View} from 'react-native';
+import PropTypes from 'prop-types';
+import _ from 'underscore';
+import styles from '../../styles/styles';
+import RenderHTML from '../RenderHTML';
+import stylePropTypes from '../../styles/stylePropTypes';
+import Text from '../Text';
+import Image from '../Image';
+
+const propTypes = {
+    /** array of image and thumbnail URIs */
+    images: PropTypes.arrayOf(
+        PropTypes.shape({
+            thumbnail: PropTypes.string,
+            image: PropTypes.string,
+        }),
+    ).isRequired,
+
+    /** optional: max number of images to show in the row if different than images length */
+    // eslint-disable-next-line react/require-default-props
+    size: PropTypes.number,
+
+    /** optional: total number of images if different than images length */
+    // eslint-disable-next-line react/require-default-props
+    total: PropTypes.number,
+
+    hoverStyle: stylePropTypes,
+};
+
+const defaultProps = {
+    hoverStyle: {},
+};
+
+function ReportActionItemImages(props) {
+    const size = props.size || props.images.length;
+    const images = props.images.slice(0, size);
+    const remaining = (props.total || props.images.length) - size;
+
+    return (
+        <View style={[styles.reportActionItemImages, props.hoverStyle]}>
+            {_.map(images, ({thumbnail, image}, index) => {
+                const isLastImage = index === props.size - 1;
+                return (
+                    <View
+                        key={image}
+                        style={[styles.reportActionItemImage, props.hoverStyle]}
+                    >
+                        {thumbnail ? (
+                            <RenderHTML
+                                html={`
+                                    <img
+                                        src="${thumbnail}"
+                                        data-expensify-source="${image}"
+                                        data-expensify-fit-container="true"
+                                        data-expensify-preview-modal-disabled="true"
+                                    />
+                            `}
+                            />
```

Comment:
I see that we are using this in at least two places. Maybe we should extract it into it's own component.